pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -409,14 +409,14 @@ in
     hardeningDisable = [ "format" ];
     env.NIX_CFLAGS_COMPILE = "-Wno-error=incompatible-pointer-types";
     patches = [
-      (fetchpatch {
+      (lib.optional (lib.versionOlder attrs.version "1.65.0") (fetchpatch {
```

Comment:
thanks. is the `"-Wno-error=incompatible-pointer-types"` still needed as well?

Reply:
No, it isn’t, but I don’t know which version cutoff to use for that one. BoringSSL was updated four times in the relevant range (v1.64.0-pre1\~41, v1.65.0-pre1\~90, v1.66.0-pre1\~18, v1.67.0-pre1\~7).

Reply:
maybe just pick 1.67.0 then or add a comment so someone can remove it easily in the future. thx.

Reply:
I determined through trial and error that the right cutoff for `-Wno-error=incompatible-pointer-types` is actually 1.68.1. Updated.